_posts/2022-11-07-why-python-package-health-metrics-matter.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: single
+title: "Why Thinking About Metrics Should be the First Thing That Any New Organization, Program or Project Does"
```

Comment:
re: the title, I totally agree with you, but I'm worried this title might be too broad for people to understand the content of the post. You don't spend the whole time telling people why they should think about metrics first, you are focused more on why metrics matter for pyOpenSci to reach its goals moving forward. I think

Maybe something like "Why I'm making metrics for package health a priority as pyOpenSci moves forward"?

Reply:
i like that. i'll change the title!